### PR TITLE
[WFCORE-4749] Bump domain management version to 11.0.0

### PR DESCRIPTION
**Pull request by yersan**
- Add KnownRelease.EAP73 as known release
- Add missing host-release types to wildlfy-config_11_0.xsd

Jira issue: https://issues.redhat.com/browse/WFCORE-4749

### CODE REVIEW

File: server/src/main/resources/schema/wildfly-config_11_0.xsd [hunk 1]
```diff
@@ -4978,6 +4978,33 @@
                             </xs:documentation>
                         </xs:annotation>
                     </xs:enumeration>
+                    <xs:enumeration value="WildFly16.0">
```
Comment (jmesnil) [inline server/src/main/resources/schema/wildfly-config_11_0.xsd]: could you also add `EAP7.3` please?

**Comment by darranl** [comment]
@jmesnil it is correct that the test is failing, this schema should not be modified except for cosmetic changes as it is already included in a release, if this is making changes to the allowed values in an enumeration then version 12 of the schema is required.

File: server/src/main/resources/schema/wildfly-config_11_0.xsd (hunk 1 above)
Comment (darranl) [reply server/src/main/resources/schema/wildfly-config_11_0.xsd]: It doesn't make sense for EAP 7.3 (which included this schema version) to have an exclude for EAP 7.3.

**Comment by yersan** [comment]
@darranl The schema is for Beta4, is it considered as released?

**Comment by darranl** [comment]
@yersan Yes this is release in EAP 7.3 so must be maintained in a compatible way now.

File: server/src/main/resources/schema/wildfly-config_11_0.xsd (hunk 1 above)
Comment (yersan) [reply server/src/main/resources/schema/wildfly-config_11_0.xsd]: EAP 7.3 comes with wildfly-core 10.x, this is for WF19

Comment (darranl) [reply server/src/main/resources/schema/wildfly-config_11_0.xsd]: EAP 7.3 includes version 11 of the schema.

**Comment by yersan** [comment]
Closing since we are going to use a different approach

**Pull request closed without merging**